.github/workflows/ci.yaml

```diff
@@ -2,6 +2,9 @@ name: CI
 
 on:
   push:
+    branches:
+    - master
+    - v2.0
```

Comment:
You should wildcard the v2.0 to v*.*

Reply:
I am going to make that v2.*, we can always add v3.* when we get there. I dont want this running on v1.x.